healthcheck.go

```diff
@@ -17,21 +18,23 @@ type healthService struct {
 }
 
 type healthConfig struct {
-	appDesc                  string
-	appSystemCode            string
-	appName                  string
-	port                     string
-	writerHealthURI          string
-	contentResolverHealthURI string
-	producer                 producer.MessageProducer
-	client                   *http.Client
+	appDesc                    string
+	appSystemCode              string
+	appName                    string
+	port                       string
+	writerHealthURI            string
+	contentResolverHealthURI   string
+	relationsResolverHealthURI string
+	producer                   producer.MessageProducer
+	client                     *http.Client
 }
 
 func newHealthService(config *healthConfig) *healthService {
```

Comment:
The gtg healthcheck should be done in parallel. There is an example in the `methode-content-placeholder-mapper` for this.

Reply:
Done.